client/reader/site-stream/index.jsx

```diff
@@ -27,55 +34,59 @@ function checkForRedirect( site ) {
 	}
 }
 
-const SiteStream = React.createClass( {
+class SiteStream extends React.Component {
 
-	getDefaultProps: function() {
-		return { showBack: true };
-	},
+	static propTypes = {
+		siteId: React.PropTypes.number.isRequired
```

Comment:
In the empty stream I am getting:
```
vendor.development.js?v=da7a8da7db:3290 Warning: Failed prop type: Invalid prop `siteId` of type `string` supplied to `SiteStream`, expected `number`.
    in SiteStream (created by LocalizedSiteStream)
    in LocalizedSiteStream
    in Provider
```

This probably isn't a regression and was always there though so I don't mind landing this without fixing the propType issue